universal-application-tool-0.0.1/app/views/dev/AzureStorageDevViewStrategy.java

```diff
@@ -0,0 +1,50 @@
+package views.dev;
+
+import static j2html.TagCreator.form;
+import static j2html.TagCreator.input;
+import static j2html.TagCreator.text;
+
+import j2html.TagCreator;
+import j2html.tags.ContainerTag;
+import services.cloud.StorageUploadRequest;
+import services.cloud.azure.BlobStorageUploadRequest;
+import views.HtmlBundle;
+import views.ViewUtils;
+
+/** Strategy class for creating a file upload form for Azure. */
+public class AzureStorageDevViewStrategy implements CloudStorageDevViewStrategy {
+
+  private static final String AZURE_STORAGE_BLOB_WEB_JAR =
+      "lib/azure__storage-blob/browser/azure-storage-blob.min.js";
+
+  @Override
+  public ContainerTag getFileUploadForm(
+      ViewUtils viewUtils, StorageUploadRequest storageUploadRequest, HtmlBundle bundle)
+      throws RuntimeException {
+    if (!(storageUploadRequest instanceof BlobStorageUploadRequest)) {
+      throw new RuntimeException(
+          "Trying to upload file to dev Azure blob storage using incorrect upload request type.");
```

Comment:
suggestion (blocking): include "azurite" in the error message to clarify e.g. "...file to azurite (Azure emulator)..."